src/richtext/richtextbuffer.cpp

```diff
@@ -7215,22 +7257,31 @@ bool wxRichTextPlainText::GetRangeSize(const wxRichTextRange& range, wxSize& siz
 
     bool haveDescent = false;
     int startPos = range.GetStart() - GetRange().GetStart();
-    long len = range.GetLength();
 
-    wxString str(m_text);
-    if (context.HasVirtualText(this))
-    {
-        if (!context.GetVirtualText(this, str) || str.Length() != m_text.Length())
-            str = m_text;
-    }
+    wxString stringChunk;
 
-    wxString toReplace = wxRichTextLineBreakChar;
-    str.Replace(toReplace, wxT(" "));
+    {
```

Comment:
Why do we need this extra block/indent?

Reply:
It is actually not needed. It is a habit of me in long functions if I need many temporary variables that are only needed at that spot. It makes reasoning about the program easier in the rest of the function. It also leads to earlier destruction of these temporary objects (it seldom matters, but it sometimes does).

The downside is that it can be confused with regular control flow blocks and it adds extra indentation.